src/main/kotlin/pet/project/app/util/profiling/LoggingProfilingConsumer.kt

```diff
@@ -0,0 +1,17 @@
+package pet.project.app.util.profiling
+
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+
+@Component
+class LoggingProfilingConsumer : ProfilingConsumer {
+    override fun accept(message: String) {
```

Comment:
```suggestion
    override fun consume(data: ProfilingData) {
```
where `ProfilingData`:
```kotlin
data class ProfilingData(
    method: String,
    duration: Duration,
) 
```